tools/appliance/systemvmtemplate/scripts/install_systemvm_packages.sh

```diff
@@ -80,10 +80,6 @@ function install_packages() {
 
   apt-get install -y python3-json-pointer python3-jsonschema cloud-init
 
-  # python2-netaddr workaround
-  wget https://github.com/shapeblue/cloudstack-nonoss/raw/main/python-netaddr_0.7.19-1_all.deb
```

Comment:
Migration of VR codebase to py3 should require us to completely get rid of netaddr/py2